src/shim/actors/miner.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019-2024 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod r#impl;
+
+use fil_actor_interface::miner::State;
+use fil_actors_shared::fvm_ipld_bitfield::BitField;
+use fvm_ipld_blockstore::Blockstore;
+
+use crate::rpc::types::SectorOnChainInfo;
+
+pub trait MinerStateExt {
```

Comment:
I feel it makes things much easier to move `fil_actors_inteface` code into `forest`

Reply:
Doesn't it make it inconsistent? I don't have a strong opinion on where this should be, but I'd rather have all this in one place and not scattered randomly.

Reply:
We could have a vote maybe. I prefer the interface in forest to 
1. avoid publishing `fil_actor_*` crates on every change.
2. avoid the shim in the current `fil_actors_interface` which does not implement `LotusJson` or `JsonSchema` and we have to re-define a same struct in `forest`

Reply:
Feel free to propose an initiative to change this. I only would like this to be in the same place to avoid wildly jumping between repos.

Reply:
@hanabi1224 I prefer the actor interface to be in forest as well.